hack/make/.build-empty-images

```diff
@@ -24,3 +29,27 @@ if ! docker image inspect emptyfs > /dev/null; then
 	)
 	rm -rf "$dir"
 fi
+
+# without c8d image store, image id is the config's id
+dangling_cfg=0df1207206e5288f4a989a2f13d1f5b3c4e70467702c1d5d21dfc9f002b7bd43
+# with c8d image store, image id is the id of manifest/manifest list.
+dangling_mfst=16d365089e5c10e1673ee82ab5bba38ade9b763296ad918bd24b42a1156c5456
+if imageNotPresent "$dangling_cfg" && imageNotPresent "$dangling_mfst"; then
+	dir="$DEST/dangling"
+	mkdir "$dir"
+	(
+		cd "$dir"
+		echo -n '{"schemaVersion":2,"manifests":[{"mediaType":"application/vnd.docker.distribution.manifest.v2+json","digest":"sha256:16d365089e5c10e1673ee82ab5bba38ade9b763296ad918bd24b42a1156c5456","size":264,"annotations":{"org.opencontainers.image.created":"2023-05-19T08:00:44Z"},"platform":{"architecture":"amd64","os":"linux"}}]}' > index.json
```

Comment:
Nit: let's use `printf` instead of `echo -n`.

Reply:
Ah, I used `printf` initially, but didn't see much advantage and changed to echo to stay consistent with the `emptyfs` 😄 

Curious - why `printf` is better in this case?

Reply:
`echo -n` isn't POSIX; technically it's fine as you'll get the builtin version in Bash given the shebang line, but it's not best practice/shellcheck should complain: https://www.shellcheck.net/wiki/SC3037

Reply:
Ah, good to know. Changing to `printf`, thanks!